internal/exec/util/util.go

```diff
@@ -15,18 +15,13 @@
 package util
 
 import (
-	"errors"
 	"os"
 	"path/filepath"
 
 	"github.com/coreos/ignition/v2/internal/log"
 	"github.com/coreos/ignition/v2/internal/resource"
 )
 
-var (
-	errEscapedMountpoint = errors.New("Symlink traversal resulted in path outside of filesystem")
```

Comment:
This and `wantsToEscape` are unused since 5fd51005bf45319bb188d19cfdc92287de9c80a0.